degov_media_gallery/js/gallery.js

```diff
@@ -34,14 +34,14 @@
       }
       $.each($images, function (k, img) {
         var $pswpItem = {
-          src: drupalSettings.degov_media_gallery.imagesDownloadLinks[k].uri,
-          w: drupalSettings.degov_media_gallery.imagesDownloadLinks[k].width,
-          h: drupalSettings.degov_media_gallery.imagesDownloadLinks[k].height
+          src: settings.degov_media_gallery.imagesDownloadLinks[k].uri,
```

Comment:
it is better to use local variable settings, because on ajax changes it will have the latest variable's value